lightclient-circuits/src/committee_update_circuit.rs

```diff
@@ -324,7 +323,7 @@ mod tests {
 
         let timer = start_timer!(|| "committee_update mock prover");
         let prover = MockProver::<Fr>::run(K, &circuit, instance).unwrap();
-        prover.assert_satisfied_par();
+        prover.assert_satisfied();
```

Comment:
Why's this changed?

Reply:
looks like it was removed from latest halo2-pse, anyway it's only useful when parallel witness assignment is used.

also it's broken in halo2-axiom https://github.com/axiom-crypto/halo2/issues/32